Methods missing from importlib.machinery.ModuleSpec.loader()

The following code:
```python3
# test.py
from importlib.util import find_spec, module_from_spec
spec = find_spec('A')
assert spec
module = module_from_spec(spec)
assert spec.loader
spec.loader.exec_module(module)
```

```python3
# A.py
print("OK")
```
produces the following ouput:
```shell
$ python3 test.py
OK

$ mypy test.py
test.py:6:1: error: "_LoaderProtocol" has no attribute "exec_module"  [attr-defined]
    spec.loader.exec_module(module)
    ^
Found 1 error in 1 file (checked 1 source file)

$ mypy --version
mypy 0.910

$ python3 --version
Python 3.8.10

$ cat /etc/issue
Ubuntu 20.04.3 LTS \n \l

$ uname -a
Linux ubuntu 5.4.0-88-generic #99-Ubuntu SMP Thu Sep 23 17:29:00 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```
I'm not sure how to fix this properly, as the current stubs for `importlib.machinery.ModuleSpec` are structured much differently than the code.

--- a/stdlib/importlib/machinery.pyi
+++ b/stdlib/importlib/machinery.pyi
@@ -2,7 +2,6 @@ import importlib.abc
 import types
 from typing import Any, Callable, Sequence
 
-# TODO: the loaders seem a bit backwards, attribute is protocol but __init__ arg isn't?
 class ModuleSpec:
     def __init__(
         self,

